tensorflow/tensorflow.bzl

```diff
@@ -1298,11 +1298,16 @@ def tf_py_wrap_cc(name,
           "//conditions:default": [":" + cc_library_name],
       }))
 
-def py_test(deps=[], **kwargs):
+def py_test(deps=[], data=[], **kwargs):
   native.py_test(
       deps=select({
           "//conditions:default": deps,
-          clean_dep("//tensorflow:no_tensorflow_py_deps"): []
+          clean_dep("//tensorflow:no_tensorflow_py_deps"): [],
+      }),
+      data = data + select({
+          "//conditions:default": [],
+          clean_dep("//tensorflow:no_tensorflow_py_deps"):
+          ["//tensorflow/tools/pip_package:build_pip_package_marker"],
```

Comment:
Can you add a comment to explain what this does? (It wasn't obvious to me, and so I'm assuming it won't be to future readers....) In particular, why does it take the marker, which is the output of `md5sum` of two targets, as `data` rather than taking those two targets as `data` directly? Is it an optimization?

Reply:
Comments added, please take a look. Thanks for the suggestion!